models/user.model.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+const generate = require("../helpers/generate");
+const userSchema = new mongoose.Schema(
+    {
+        fullName: String,
+        email: String,
+        password: String,
+        tokenUser: {
+            type: String,
+            default: generate.generateRandomString(30)
+        },
```

Comment:
This is not a great way to store user tokens. It's ok for now. We can improve it later.